uPortal-api/uPortal-api-rest/src/main/java/org/apereo/portal/layout/dlm/remoting/UpdatePreferencesServlet.java

```diff
@@ -695,21 +696,22 @@ public ModelAndView removeFavorite(
                             favoritePortlets, functionalName);
 
             String resp = new String();
-            // String nodeId = new String();
 
             for (IUserLayoutChannelDescription deleteNode : favoritesToDelete) {
                 if (deleteNode != null && deleteNode instanceof UserLayoutChannelDescription) {
                     UserLayoutChannelDescription channelDescription =
                             (UserLayoutChannelDescription) deleteNode;
                     try {
                         if (!ulm.deleteNode(channelDescription.getChannelSubscribeId())) {
+
                             logger.warn(
-                                    "Error deleting the node"
-                                            + channelId
-                                            + "from favorites for user "
-                                            + (upm.getPerson() == null
+                                    MessageFormat.format(
```

Comment:
You don't need to use MessageFormat, SLF4J permit to replace variable in the order of argument passed. As example see on line https://github.com/uPortal-Project/uPortal/blob/d9fdb4523a5ff7bd8a9c791c6377966dba267233/uPortal-api/uPortal-api-rest/src/main/java/org/apereo/portal/layout/dlm/remoting/UpdatePreferencesServlet.java#L233

But anyway thanks !

Reply:
Thanks for the tip, I removed MessageFormat.